crates/providers/blobstore-s3/wit/deps/blobstore/blobstore.wit

```diff
@@ -0,0 +1,364 @@
+package wasmcloud:blobstore;
+
+/// This interface represents the functions necessary
+/// to implement the [blobstore wasmCloud interface](https://github.com/wasmCloud/interfaces/blob/main/blobstore/blobstore.smithy)
+///
+/// Derived from https://github.com/wasmCloud/interfaces/blob/f020c93d4cacd50318301f686e2f059a15862e1e/blobstore/blobstore.smithy
+interface blobstore {
+    type container-id = string;
+
+    /// For Smithy, timestamps mean time between January 1, 0001 CE to 23:59:59.999 and
+    /// December 31, 9999 CE, with millisecond granularity. While Smithy (v2.0) intentionally
+    /// does not constrain serialization/deserialiation semantics, we must take a stance here.
+    ///
+    /// This timestamp represents - represents absolute time in UTC,
+    /// as non-leap seconds and nanoseconds since the UNIX EPOCH
+    record timestamp {
+      /// The number of non-leap seconds since UNIX EPOCH in UTC
+      /// TODO(vadossi-cosmonic): this is a break with the previous Timestamp type which used an i64
+      sec: u64,
```

Comment:
Is there anything to be todone here?

Reply:
Nope, this I thought would be a problem but it turns out it wasn't